zilencer/management/commands/populate_db.py

```diff
@@ -1174,25 +1176,25 @@ def bulk_create_reactions(all_messages: List[Message]) -> None:
     Reaction.objects.bulk_create(reactions)
 
 
-def choose_date_sent(num_messages: int, tot_messages: int, threads: int) -> datetime:
+def choose_date_sent(num_messages: int, tot_messages: int, starting_n_days_ago: int, threads: int) -> datetime:
```

Comment:
I figured sending just this one variable through is probably enough, unless we have a desire to modify the 3 or 1 numbers later.